### PR TITLE
1. Update cnf-common.sh

### PR DESCRIPTION
**Pull request by zhangyi733**
2. Add more checkings in checkstyle.xml

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -39,6 +39,18 @@ checkstyle {
     configDir rootProject.file('config/checkstyle')
     configFile rootProject.file('config/checkstyle/checkstyle.xml')
```
Comment (bhavikkumar) [inline build.gradle]: We probably don't need the two lines above since they are the default location.

File: build.gradle [hunk 2]
```diff
@@ -39,6 +39,18 @@ checkstyle {
...
+        source = sourceSets.main.allSource
+    }
+
+    configurations {
+        checkstyle
+    }
+
+    dependencies {
+        checkstyle "com.puppycrawl.tools:checkstyle:8.13"
+    }
```
Comment (bhavikkumar) [inline build.gradle]: Do we need the configurations and dependencies sections? They aren't in the plugins docs so not sure what they are doing in this context.

File: config/checkstyle/checkstyle.xml [hunk 3]
```diff
@@ -64,7 +65,6 @@
     </module>
     <module name="NeedBraces"/>
     <module name="LeftCurly">
-      <property name="maxLineLength" value="150"/>
```
Comment (bhavikkumar) [inline config/checkstyle/checkstyle.xml:67]: What is the max line length now?

Comment (zhangyi733) [reply config/checkstyle/checkstyle.xml:67]: It's still 150, but not in `LeftCurly` module. You can refer to module 
` <module name="LineLength">
      <property name="max" value="150"/>
      <property name="ignorePattern"
        value="^package.*|^import.*|a href|href|http://|https://|ftp://"/>
    </module>`

File: build.gradle (hunk 2 above)
Comment (zhangyi733) [reply build.gradle]: Agreed. We have removed this one.

File: build.gradle (hunk 1 above)
Comment (zhangyi733) [reply build.gradle]: Agreed. We have removed the second line.

**Review by alphafoobar — APPROVED** [review]

**Review by bhavikkumar — APPROVED** [review]

**Pull request merged**